cmd/codegen/gengo.go

```diff
@@ -38,7 +38,8 @@ var replace = map[string]string{
 	"igGetStyle":              "CurrentStyle",
 	"igGetMouseCursor":        "CurrentMouseCursor",
 	"ImAxis":                  "PlotAxisEnum",
-	"GetItemID":               "GetItemByLabelID",
+	"GetItem_ID":              "ItemByID",
+	"GetItemID":               "ItemID",
```

Comment:
I think it doesn't need to be here - will be updated changed

Reply:
(I mean `	"GetItemID":               "ItemID",`)

Reply:
Ok. Will remove this

Reply:
Done.